tidb-cloud/tidb-cloud-auditing.md

```diff
@@ -174,6 +174,78 @@ In the TiDB Cloud console, go back to the **Enable Database Audit Logging** dial
 > - After enabling audit logging, if you make any new changes to the bucket URI or location, you must click **Test Connection** again to verify that TiDB Cloud can connect to the bucket. Then, click **Enable** to apply the changes.
 > - To remove TiDB Cloud's access to your GCS bucket, delete the trust policy granted to this cluster in the Google Cloud console.
 
+### Enable audit logging for Azure
+
+To enable audit logging for Azure, take the following steps:
+
+#### Step 1. Create an Azure storage account
+
+Create an Azure storage account in your organization's Azure subscription as the destination to which TiDB Cloud writes the database audit logs.
+
+For more information, see [Create an Azure storage account](https://learn.microsoft.com/en-us/azure/storage/common/storage-account-create?tabs=azure-portal) in Azure documentation.
+
+#### Step 2. Configure Azure Blob Storage access
+
+1. In the [Azure portal](https://portal.azure.com/), create a container used for storing database audit logs.
+
+    1. In the left navigation pane of the Azure portal, click **Storage Accounts**, and then click the storage account for storing database audit logs.
+
+        > **Tip:**
+        >
+        > If the left navigation pane is hidden, click the menu button in the upper-left corner to toggle its visibility.
+
+    2. In the navigation pane for the selected storage account, click **Data storage > Containers**, and then click **+ Container** to open the **New container** pane.
+
+    3. In the **New container** pane, enter a name for your new container, set the anonymous access level (the recommended level is **Private**, which means no anonymous access), and then click **Create**. The new container will be created and displayed in the container list in a few seconds.
+
+2. Get the URL of the target container.
+
+    1. In the container list, select the target container, click **...** for the container, and then select **Container properties**.
+    2. On the displayed properties page, copy the **URL** value for later use, and then return to the container list.
+
+3. Generate a SAS token for the target container.
+
+    1. In the container list, select the target container, click **...** for the container, and then select **Generate SAS**.
+    2. In the displayed **Generate SAS** pane, select **Account key** for **Signing method**.
+    3. In the **Permissions** drop-down list, select **Read**, **Write**, and **Create** to allow writing audit log files.
+    4. In the **Start** and **Expiry** fields, specify a validity period for the SAS token.
+
+        > **Note:**
+        >
+        > - Audit logging is a long-term feature that requires a SAS token with a sufficiently long validity period. However, longer validity increases the risk of token leakage. For security, it is recommended to replace your SAS token every six to twelve months.
```

Comment:
The word "long-term feature" seems odd. I think we can say audit logging feature continuously writes audit log to the storage account, so it requires a SAS token with a sufficiently long validity period.